media-sound/mikmod/mikmod-3.2.8.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A module player for Unix"
+DESCRIPTION="libmikmod supports many formats, including mod, s3m, it, and xm."
```

Comment:
whoops :) missed that one :) Aside from that most of the description is the same :)